app/app/custom-transport.ts

```diff
@@ -0,0 +1,197 @@
+
+import type { DescMessage, DescMethod, MessageInitShape, MessageShape } from "@bufbuild/protobuf";
+
+import type { ContextValues, UnaryRequest } from "@connectrpc/connect";
+import { Code, ConnectError, createContextValues } from "@connectrpc/connect";
+import type { Transport, UnaryResponse } from "@connectrpc/connect";
+import {
```

Comment:
Create a custom transport because of limited React Native fetch functionality. Only supports unary procedures.